install/upgrade/upgrade_core_db.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE cws_worker ADD now_post int(11) NOT NULL;
```

Comment:
What is this column name?  Does it match the real column name?

Reply:
Corrected, the column name to `max_num_running_procs`